src/elements/OcButton.vue

```diff
@@ -4,15 +4,14 @@
     :href="href"
     :to="to"
     :type="submit"
-    :text="text"
```

Comment:
let's keep v-text here - see https://vuejs.org/v2/api/#v-text .... maybe somthing to consider in different places as well ....

Reply:
I'm not sure I understand how is this supposed to help. If you include it with v-text then you need to write again getText - right? Which in the end I think is the same amount of work as to write it into its own slot.

Reply:
The textContent can also be taken from the inner node I I understand correctly. That is my point